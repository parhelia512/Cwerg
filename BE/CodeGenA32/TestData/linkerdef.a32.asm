# linkerdef

.mem dummy 4 RW
.data 4 "0"

# ========================================
.fun main NORMAL [S32] = []
.bbl start
    ld.mem x:U32 = dummy 0

    pusharg x
    bsr print_x_ln

    lea.mem a:A32 $$rw_data_end 0
    bitcast x a

    pusharg x
    bsr print_x_ln

    pusharg 0:S32
    ret
