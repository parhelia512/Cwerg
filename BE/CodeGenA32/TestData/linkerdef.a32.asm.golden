30303030
30240
